django/utils/dateparse.py

```diff
@@ -51,6 +51,18 @@
 )
 
 
+postrges_interval_re = re.compile(
```

Comment:
This regex is very similar to the standard one, but I'm not sure if it can (or should) be merged with that one.

For instance, it's legal to have a `-` on any of the parts of a standard duration, but that doesn't happen on a postgres interval: you only get a maximum of two (one on the number of days, and one on the `h:m:s.ms` section).